ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/QueryDescriptionTest.java

```diff
@@ -15,173 +15,136 @@
 
 package io.confluent.ksql.rest.entity;
 
-import static org.easymock.EasyMock.expect;
-import static org.easymock.EasyMock.mock;
-import static org.easymock.EasyMock.niceMock;
-import static org.easymock.EasyMock.replay;
 import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
 
-import io.confluent.ksql.function.FunctionRegistry;
-import io.confluent.ksql.logging.processing.ProcessingLogContext;
+import com.google.common.collect.ImmutableSet;
 import io.confluent.ksql.metastore.KsqlStream;
 import io.confluent.ksql.metastore.KsqlTopic;
-import io.confluent.ksql.planner.plan.OutputNode;
-import io.confluent.ksql.planner.plan.PlanNodeId;
-import io.confluent.ksql.planner.plan.StructuredDataSourceNode;
+import io.confluent.ksql.physical.LimitHandler;
 import io.confluent.ksql.query.QueryId;
+import io.confluent.ksql.rest.entity.SchemaInfo.Type;
 import io.confluent.ksql.serde.DataSource;
 import io.confluent.ksql.serde.json.KsqlJsonTopicSerDe;
-import io.confluent.ksql.services.ServiceContext;
-import io.confluent.ksql.structured.SchemaKStream;
-import io.confluent.ksql.util.KsqlConfig;
 import io.confluent.ksql.util.PersistentQueryMetadata;
-import io.confluent.ksql.util.QueryIdGenerator;
 import io.confluent.ksql.util.QueryMetadata;
 import io.confluent.ksql.util.QueuedQueryMetadata;
 import io.confluent.ksql.util.timestamp.MetadataTimestampExtractionPolicy;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.function.Consumer;
 import org.apache.kafka.common.serialization.Serdes;
-import org.apache.kafka.connect.data.Field;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.apache.kafka.streams.KafkaStreams;
-import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.Topology;
 import org.apache.kafka.streams.TopologyDescription;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class QueryDescriptionTest {
+
   private static final Schema SCHEMA =
       SchemaBuilder.struct()
           .field("field1", SchemaBuilder.int32().build())
           .field("field2", SchemaBuilder.string().build())
           .build();
+
+  private static final List<FieldInfo> EXPECTED_FIELDS = Arrays.asList(
+      new FieldInfo("field1", new SchemaInfo(Type.INTEGER, null, null)),
+      new FieldInfo("field2", new SchemaInfo(Type.STRING, null, null)));
+
   private static final String STATEMENT = "statement";
+  private static final Map<String, Object> STREAM_PROPS = Collections.singletonMap("k1", "v1");
```

Comment:
```suggestion
  private static final Map<String, Object> STREAMS_PROPS = Collections.singletonMap("k1", "v1");
```
To be consistent with our naming elsewhere.

Reply:
done.